src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -9,9 +9,6 @@
 .root {
   composes: hds-typography-body-300 from global;
 
-  --permalink-scroll-offset: 24px;
-  --height: calc(var(--sticky-bars-height) + var(--permalink-scroll-offset));
```

Comment:
Intent is to use the global CSS helper class `.g-offset-scroll-margin-top`. This class is set up in `pages/style.css`, and is now used by our MDX heading components.